Content.Shared/FixedPoint/FixedPoint2.cs

```diff
@@ -234,6 +234,11 @@ public static FixedPoint2 Clamp(FixedPoint2 reagent, FixedPoint2 min, FixedPoint
                    _value == unit._value;
         }
 
+        public bool EqualsApprox(FixedPoint2 other, FixedPoint2 tolerance)
+        {
+            return Math.Abs(_value - other._value) < tolerance._value;
+        }
```

Comment:
is this useful at all? the values already arent going to differ by any more than `0.01`

Reply:
I remember there was a reason I needed this at the time, but don't remember why. The only usage has since been removed and I guess I just forgot to remove this as well. Removing now